packages/kbn-test/src/functional_tests/cli/start_servers/cli.js

```diff
@@ -17,16 +17,25 @@
  * under the License.
  */
 
-export const functional = {
-  options: {
-    logLevel: 'debug',
-    configFile: require.resolve('../../test/functional/config.js')
-  }
-};
+import chalk from 'chalk';
+import getopts from 'getopts';
+import { startServers } from '../../tasks';
+import { processOptions, displayHelp } from './args';
+
+/**
+ * Start servers
+ * @param {string} defaultConfigPath Path to config to use
```

Comment:
__nit:__ please mark this parameter as optional if it's optional.

Same comment applies to `cli.js` in `run_tests`.

Reply:
done